astro/terraform/version_check.go

```diff
@@ -31,3 +31,9 @@ func VersionMatches(v *version.Version, versionConstraint string) bool {
 	}
 	return constraint.Check(v)
 }
+
+// StringVersionMatches returns whether or not the version passed as string matches the constraint.
+// See VersionMatches for more info.
+func StringVersionMatches(v string, versionConstraint string) bool {
```

Comment:
If the function is only used for testing, we should make it private and probably move it to the test file.

Reply:
Agreed. I moved this to integration_test.go. Pls take another look.